src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,91 @@
+.item__product-card {
+  width: 200px;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__description {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.product-card__product-code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: var(--font-weight-main);
+  line-height: 14px;
+  color: var(--secondary-color);
+}
+
+.product-card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.product-card__reviews-count {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-color);
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.product-card__price-name {
+  font-size: 12px;
+  line-height: 16px;
+  color: var(--secondary-color);
+}
+
+.product-card__price-tag {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.product-card__buy-button {
+  display: flex;
+  font-size: 14px;
+  line-height: 40px;
+  font-weight: 700;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+.product-card__buy-link {
+  align-self: center;
+  transition: 0.3s;
```

Comment:
don't animate all properties, only that you need

Reply:
![image](https://user-images.githubusercontent.com/99331793/203063404-c95d20e2-275a-4cbc-af85-afa21a5236b2.png)
not fixed